src/appendices/phenotype.md

```diff
@@ -0,0 +1,436 @@
+# Tabular phenotypic data guidelines
+
+This appendix is a collection of guidelines and examples
+for curating well-organized tabular phenotypic data.
+
+## Guidelines
+
+These guidelines are intended to improve the organization and clarity of
+tabular phenotypic data like the participants file, sessions file,
+and phenotypic and assessment data.
+
+They are recommendations and are by default ignored during validation.
+You can make them mandatory during validation by setting the
+[`AdditionalValidation` key](../modality-agnostic-files/dataset-description.md#additional-validation)
+contains `"Phenotype"` in the `dataset_description.json`.
+
+### 1. Aggregate data across sessions
+
+Aggregate participant information across all sessions into one tabular TSV file per
+measurement or phenotypic assessment and store this file in the `/phenotype` directory.
+Demographic information is a special case and  MUST be aggregated
+in the `participants.tsv` file at the root level of the dataset.
+It is RECOMMENDED to use the `age` column in the `participants.tsv` file
```

Comment:
Theoretically we could validate the appropriate age being used in each session based on the relative acq_times if present but I don't think its worth the effort. Maybe monotonically increasing age like schema.rules.checks.mri.VolumeTimingNotMonotonicallyIncreasing would be a compromise?

Reply:
I agree it's not worth the effort. That and we can't rely on the age monotonically increasing as some sessions may be close enough to not affect the reported age.